docs/src/components/pricing/PricingTable.tsx

```diff
@@ -79,84 +86,127 @@ interface PlanPriceProps {
 export function PlanPrice(props: PlanPriceProps) {
   const { plan } = props;
 
+  const { licenseType } = useLicenseType();
+  const annual = licenseType === 'Annual';
+
   if (plan === 'community') {
     return (
-      <Box sx={{ display: 'flex', alignItems: 'center', mt: 1 }}>
-        <Typography variant="h4" component="div" fontWeight="bold" color="success.600">
-          $0
-        </Typography>
-        <Box sx={{ width: 5 }} />
-        <Typography variant="body2" color="text.secondary" sx={{ mt: '4px' }}>
-          – free forever
+      <React.Fragment>
+        <Box sx={{ display: 'flex', alignItems: 'center', justifyContent: 'center', mt: 1, mb: 4 }}>
+          <Typography
+            variant="h3"
+            component="div"
+            fontWeight="bold"
+            color="success.600"
+            sx={{ mt: 6 }}
```

Comment:
Adjusting margin to match alignment with the paid plans.

```suggestion
            sx={{ mt: 4.5 }}
```